app/lib/actions/katello/content_view_version/incremental_update.rb

```diff
@@ -30,17 +31,26 @@ def plan(old_version, environments, options = {})
           validate_environments(environments, old_version)
 
           new_minor = old_version.content_view.versions.where(:major => old_version.major).maximum(:minor) + 1
+          if SmartProxy.pulp_master.pulp3_support?(old_version.repositories.first) && is_composite
```

Comment:
i saw something similar above, checking that the first repo in the cvv is supported under pulp3.  Whats the reason for that?  What would happen if i published a mixed set of content (file under pulp2, rpm under pulp3)?